test/hwdb-test.sh

```diff
@@ -32,7 +32,7 @@ D=$(mktemp --directory)
 trap "rm -rf '$D'" EXIT INT QUIT PIPE
 mkdir -p "$D/etc/udev"
 ln -s "$ROOTDIR/hwdb" "$D/etc/udev/hwdb.d"
-err=$("$SYSTEMD_HWDB" update --root "$D" 2>&1 >/dev/null)
+err=$("$SYSTEMD_HWDB" update --root "$D")
```

Comment:
This breaks the test actually, as this stops being able to detect if there is any stderr. If we don't expect regular stdout, then just put back the `2>&1`, and maybe rename the variable  to `out`?